redirects-list.json

```diff
@@ -82,14 +82,16 @@
 
   "^/doc/vs-code-extension$                                                               https://marketplace.visualstudio.com/items?itemName=Iterative.dvc 302",
 
-  "^/doc/studio/view-settings$                                                            /doc/studio/user-guide/projects/project-settings",
-  "^/doc/studio/user-guide/explore-experiments$                                           /doc/studio/user-guide/projects/explore-experiments",
-  "^/doc/studio/user-guide/visualize-experiments$                                         /doc/studio/user-guide/projects/visualize-experiments",
-  "^/doc/studio/user-guide/run-experiments$                                               /doc/studio/user-guide/projects/run-experiments",
-
-  "^/doc/studio/user-guide/views/create-view$                                             /doc/studio/user-guide/projects/create-project",
-  "^/doc/studio/user-guide/views/view-settings$                                           /doc/studio/user-guide/projects/project-settings",
-  "^/doc/studio/user-guide/views/(.*)$                                                    /doc/studio/user-guide/projects/$1",
+  "^/doc/studio/view-settings$                                                            /doc/studio/user-guide/views/view-settings",
```

Comment:
Why are we not directly redirecting it to the final version instead of multiple redirects 🤔?
i.e
 ```
"^/doc/studio/view-settings$     /doc/studio/user-guide/projects/project-settings 302",
```

Reply:
Because the 301 redirect to `/doc/studio/user-guide/views/` has been published for a while already (recently removed [in #3728](https://github.com/iterative/dvc.org/pull/3728/files#diff-a6828b19e9794c1fbb6799ea95263fb7487b676d054932b5f6e3b315eb12a6a2)) and as stated in the OP, we shouldn't change existing (301) redirects. That's because 301s get cached indefinitely by browsers so once we publish them they should be considered "final" (which I think means we need to wait about a year before removing them), especially if they've been published for a while as is the case here.

> BTW this list needs an urgent clean up I think...

Reply:
p.s. in this case maybe the cached redirects did get caught by the `^/doc/studio/user-guide/views/(.*)$` redirect but still, as a best practice we should avoid changing existing 301 redirects until after a long time has passed, I think. Up to you @yathomasi lmk what to do here please.

Reply:
I merged this to get moving (needed for #3621) but lmk what you think @yathomasi and we can address redirects separately (see https://github.com/iterative/dvc.org/issues/3776)

Reply:
Umm, I think the existing cached 301 redirects will still be working even if we point the redirects directly to the final version.
Existing:
`^/doc/studio/view-settings$` -> `/doc/studio/user-guide/views/view-settings` 
`/doc/studio/user-guide/views/view-settings` -> `/doc/studio/user-guide/projects/project-settings 302`

Proposed:
`^/doc/studio/view-settings$` -> `/doc/studio/user-guide/projects/project-settings 302`
`/doc/studio/user-guide/views/view-settings` -> `/doc/studio/user-guide/projects/project-settings 302`

So, the link `^/doc/studio/view-settings$` will not have to jump into multiple redirects and the previous redirects will still be working.